### PR TITLE
Updated descriptions in webapplicationfirewall.json

### PR DESCRIPTION
**Pull request by EvgenyAgafonchikov**
NetworkValidation CI task currently checks descriptions code style (first letter casing).
This fixes potential failures in the next PRs.
Also, added trailing dots to update all at once.
[Related job](https://dev.azure.com/azure-sdk/public/_build/results?buildId=31164)

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/webapplicationfirewall.json [hunk 1]
```diff
@@ -282,25 +282,25 @@
       ]
     },
     "WebApplicationFirewallPolicyPropertiesFormat": {
-      "description": "Defines web application firewall policy properties",
+      "description": "Defines web application firewall policy properties.",
       "properties": {
         "policySettings": {
-          "description": "Describes  policySettings for policy",
+          "description": "Describes  policySettings for policy.",
```
Comment (number213) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/webapplicationfirewall.json]: Extra space after "Describes"

**Review by anton-evseev — CHANGES_REQUESTED** [review]

File: specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/webapplicationfirewall.json [hunk 2]
```diff
@@ -282,25 +282,25 @@
...
+      "description": "Defines web application firewall policy properties.",
       "properties": {
         "policySettings": {
-          "description": "Describes  policySettings for policy",
+          "description": "Describes policySettings for policy.",
           "$ref": "#/definitions/PolicySettings"
         },
         "policyDetailSettings": {
-          "description": "Describes policyDetailSettings for policy",
+          "description": "Describes policyDetailSettings for policy.",
```
Comment (number213) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/webapplicationfirewall.json]: Please make "policyDetailSettings" more human-readable, e.g. "policy detail settings"

File: specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/webapplicationfirewall.json [hunk 3]
```diff
@@ -282,25 +282,25 @@
       ]
     },
     "WebApplicationFirewallPolicyPropertiesFormat": {
-      "description": "Defines web application firewall policy properties",
+      "description": "Defines web application firewall policy properties.",
       "properties": {
         "policySettings": {
-          "description": "Describes  policySettings for policy",
+          "description": "Describes policySettings for policy.",
```
Comment (number213) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/webapplicationfirewall.json]: Please make "policySettings" more human-readable, e.g. "policy settings"

File: specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/webapplicationfirewall.json [hunk 4]
```diff
@@ -567,10 +567,10 @@
       }
     },
     "ManagedRuleSets": {
-      "description": "Defines ManagedRuleSets - array of managedRuleSet",
+      "description": "Defines ManagedRuleSets - array of managedRuleSet.",
```
Comment (number213) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/webapplicationfirewall.json]: description -> "Array of managed rule sets"

**Review by anton-evseev — CHANGES_REQUESTED** [review]

**Comment by EvgenyAgafonchikov** [comment]
@venkatsvpr, I've discovered [this item](https://github.com/Azure/azure-rest-api-specs/pull/5965/files#diff-a369a92061eba6d5f1ece36a6c9309a8R591) only during updating descriptions. It seems to have predefined values, so shouldn't we add enum there?
Please also review other changes and confirm if they look good.

**Review by dsgouda — APPROVED** [review]
LGTM, will merge once all reviewers approve

**Comment by EvgenyAgafonchikov** [comment]
Closing as the change is in progress of reverting now.
@venkatsvpr, feel free to reuse any part of these changes in the new PR

**Pull request closed without merging**